sled-agent/Cargo.toml

```diff
@@ -46,6 +46,7 @@ macaddr.workspace = true
 mg-admin-client.workspace = true
 nexus-client.workspace = true
 nexus-config.workspace = true
+nexus-types.workspace = true
```

Comment:
This feels like potentially a big deal...but seems fine!  (I say that because `nexus_types` I _think_ is mostly implementation details of Nexus and it seems a little sketchy to leak that outside?  But it doesn't seem worth creating a separate package for this right now.)

Come to think of it, it seems like this shouldn't be necessary because RSS would be using a progenitor-generated type.  Is this necessary because our internal Nexus client uses `replace` to use this type instead of generating one?  (And if I recall correctly, that's because we have functionality like `diff` that we want to use in omdb.  Ugh.)

Reply:
Yes, it's the `replace` directive on `Blueprint` specifically that causes this. That is kind of a dicey thing to be `replace`ing, especially as we're actively changing it. :-/